contentful_management/ui_extension.py

```diff
@@ -21,15 +22,16 @@ class UIExtension(Resource):
 
     def __init__(self, item, **kwargs):
         super(UIExtension, self).__init__(item, **kwargs)
-        self.extension = item.get('extension', {})
+        self.extension = deepcopy(item.get('extension', {}))
 
     @property
     def source(self):
-        return self.extension.get('src', '')
+        return self.extension.get('src', '') or self.extension.get('srcdoc', '')
```

Comment:
That feels rather odd. Why can't we have two properties for two different things?